README.md

```diff
@@ -0,0 +1,285 @@
+# **py-sc-client** #
+
+Python implementation of client for [communication with sc-server](http://ostis-dev.github.io/sc-machine/http/websocket/). This module is tested on Python 3.8+.
```

Comment:
```suggestion
The Python implementation of the client for [communication with sc-server](http://ostis-dev.github.io/sc-machine/http/websocket/). This module is tested on Python 3.8+.
```

Reply:
It's probably not a good idea to provide links to documentation from another organisation. That said, we didn't host ours on GitHub Pages yet

Reply:
> It's probably not a good idea to provide documentation links from another organisation. That said, we didn't host ours on GitHub Pages yet

I agreed, will be changed to README with protocol description of ostis-ai organisation